recipes/ldas-tools-ldasgen-swig/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "ldas-tools-ldasgen-swig" %}
+{% set version = "2.6.6" %}
+{% set sha256 = "721f6eaaec5ad5892fab1e86fa9f7fd53f4521b105d151f21fc38ae22ded41cb" %}
+{% set ldas_tools_cmake_version = "1.0.5" %}
+{% set ldas_tools_al_version = "2.6.2" %}
+{% set ldas_tools_ldasgen_version = "2.6.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.6
+    - pkg-config  # [not win]
+    - make  # [not win]
+    - swig
+  host:
+    - ldas-tools-cmake >={{ ldas_tools_cmake_version }}
+    - ldas-tools-ldasgen >={{ ldas_tools_ldasgen_version }}
+
+outputs:
+  - name: ldas-tools-ldasgen-swig
+    script: install-interface.sh
+    build:
+      activate_in_script: True
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake >=3.6
+        - pkg-config  # [not win]
+        - make  # [not win]
+        - swig
+      host:
+        - ldas-tools-cmake >={{ ldas_tools_cmake_version }}
+        - ldas-tools-ldasgen >={{ ldas_tools_ldasgen_version }}
+      run:
+        - ldas-tools-ldasgen >={{ ldas_tools_ldasgen_version }}
+    test:
+      commands:
+        - test -f ${PREFIX}/include/genericAPI/LDASlogging.i  # [not win]
+    about:
+      home: https://wiki.ligo.org/Computing/DASWG/LDASTools
+      dev_url: https://git.ligo.org/ldastools/LDAS_Tools
+      license: GPLv3
+      license_family: GPL
+      license_file: COPYING
+      summary: SWIG interface for the LDAS Tools generic library
+  - name: python-ldas-tools-ldasgen
+    script: install-python.sh
+    build:
+      activate_in_script: true
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake >=3.6
+        - pkg-config  # [not win]
+        - make  # [not win]
+        - swig
+      host:
+        - ldas-tools-cmake >={{ ldas_tools_cmake_version }}
+        - ldas-tools-ldasgen >={{ ldas_tools_ldasgen_version }}
```

Comment:
One new feature in conda-build are run exports. See the docs here: https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#export-runtime-requirements. 

This might simplify maintenance of these tools. You could add a run export to `ldas-tools-ldasgen` and then anytime it is in the host requirements, it will always get added to the run requirements.

Reply:
I personally think its better to have things be more explicit rather than less, but whatever's the 'standard', I'm happy to adhere to.